.travis.yml

```diff
@@ -1,25 +1,30 @@
 language: php
 
-matrix:
-    include:
-        - php: hhvm
-        - php: 5.3
-        - php: 5.4
-        - php: 5.5
-        - php: 5.6
-        - php: nightly
-    allow_failures:
-        - php: hhvm
-        - php: nightly
-    fast_finish: true
-
 sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
 
-install: composer install -n
+matrix:
+  include:
+    - php: 5.3
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 5.6
+    - php: 5.6
+      env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' 
+    - php: hhvm
+    - php: nightly
+  allow_failures:
+    - php: hhvm
+    - php: nightly
+  fast_finish: true
+
+install:
+  - composer self-update -q
```

Comment:
and I would not use ``-q``. If there is an issue during the self-update, we want to see it. and if everything works fine, the output will be collapsed by Travis by default anyway